data/ecosystems/m/moonbeans.toml

```diff
@@ -3,7 +3,7 @@ title = "Moonbeans"
 
 sub_ecosystems = []
 
-github_organizations = []
+github_organizations = ["https://github.com/m00nbeans"]
```

Comment:
unfortunately this is not an org link. Instead, it's a user link. Please remove it and add the repos that correspond individually.